### PR TITLE
5.2.1 version history

### PR DESCRIPTION
**Pull request by hflynn**
Autogen build needs to run before this will be displayed in staging docs but the formatting should be fine.

### CODE REVIEW

**Comment by jburel** [comment]
@hflynn: auto-gen should be up-to-date. I have closed my PR and remove the link

**Comment by hflynn** [comment]
Ah, merge build is defaulting to building without the autogen branch, that'll be why it isn't on staging yet. Re-running.

**Comment by hflynn** [comment]
Hmmm @sbesson the build was red with the autogen merge build included -https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-merge-docs/160/ I think we still have an issue with the install scripts

**Comment by hflynn** [comment]
Now up on staging https://www.openmicroscopy.org/site/support/omero5.2-staging/users/history.html

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+System administrator updates include:
+
+-  improved installation documentation, including walkthroughs with scripts
+-  OMERO.web deployment fixes
+
+Developer updates include:
+
+-  OMERO Javadocs now link to the relevant version of Bio-Formats Javadocs for
+   inherited methods
+-  Clean-up of server dependencies
```
Comment (hflynn) [inline history.txt]: Spotted this unneeded capital BTW, will push a commit once I've gathered any further comments.

Comment (jburel) [inline history.txt]: ``installation documentation and scripts.` `
Will people not be confused and think we are talking about omero scripts (like export_as_tiff)

**Comment by jburel** [comment]
``minimal-omero-client clean-up`` => and ``pom-omero-client``

**Comment by jburel** [comment]
``Activities fixes`` PR did not make it cf https://github.com/openmicroscopy/openmicroscopy/pull/4364

**Comment by hflynn** [comment]
Wasn't entirely sure what you meant about the pom-minimal-client, hopefully I guessed right!

**Comment by jburel** [comment]
@hflynn: your guess was good ;-)

**Pull request merged**